taigit/src/actions/taigaActions.js

```diff
@@ -28,3 +32,9 @@ export const grabSprintStats = (infoForApiCall) => dispatch => {
       dispatch({type: GET_SPRINT_STATS, payload: sprintStats});
     });
 }
+
```

Comment:
Same comment as above. Moving hardcoded value to passed variable.

Reply:
This is not part of this task, I will revert the changes.. I was doing code walk through and did changes to understand the flow. :P